types/react-notification-system/index.d.ts

```diff
@@ -69,7 +61,7 @@ declare namespace NotificationSystem {
         ActionWrapper?: WrapperStyle | undefined;
     }
 
-    export interface Attributes extends React.ClassAttributes<System> {
```

Comment:
Props can't determine the ref type for class components. Needed to move code around so that NotificationSystem correctly extends a React.Component.